pkg/util/gpu/common.go

```diff
@@ -0,0 +1,48 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
please update the CODEOWNERS file to get the ownership of `/pkg/util/gpu`

Reply:
Gotcha we can take ownership of this directory.